### PR TITLE
feature/ST-350-hide-cvv-field-when-card-is-recognized-as-piba

### PR DESCRIPTION
**Pull request by pgs-gmartowski**

### CODE REVIEW

File: src/models/Card/Card.ts [hunk 1]
```diff
@@ -14,7 +14,37 @@ import { ICardDetails } from './ICard';
...
+    document.getElementById(inputId).classList.remove(Card.ERROR_CLASS);
+    document.getElementById(messageId).textContent = '';
+  }
+
+  /**
+   * Set disable attribute to given input.
+   * @param inputId
+   * @private
+   */
+  private static _disableInput(inputId: string) {
```
Comment (james-simpson-ST) [inline src/models/Card/Card.ts]: I've noticed coveralls is complaining on this branch, I guess more unit tests are required to test more of this functionality

File: src/models/Card/Card.ts [hunk 2]
```diff
@@ -150,15 +183,19 @@ class Card extends Utils {
   }
 
   /**
-   * Sets proper security code placeholder and security code value, depends on if card number is Amex or not
+   * Sets proper security code placeholder and security code value, depends on if card number is AMEX or not
    * (generally if it's "flippable" or not). List of "non-flippable" cards is defined in this class
    * `${NOT_FLIPPED_CARDS}.
    * @private
    */
   private _setSecurityCode() {
```
Comment (james-simpson-ST) [inline src/models/Card/Card.ts]: Make sure we cover this in unittest

File: src/models/Card/Card.ts (hunk 1 above)
Comment (pgs-gmartowski) [reply src/models/Card/Card.ts]: true, I'll check which lines are not covered

File: src/models/Card/Card.ts (hunk 2 above)
Comment (pgs-gmartowski) [reply src/models/Card/Card.ts]: done

**Comment by pgs-gmartowski** [comment]
@james-simpson-ST I think now we only need to adjust / add some automated tests when security code field 'disappears'

**Pull request merged**